### ISSUE TITLE
[BUG] PF1e Assign XP is being weird. 

### ISSUE THREAD

**Issue by KyleDornez** [ironmonk88/monks-tokenbar#450]
I don't know if it's a bug or I'm doing something wrong, but since the core system of PF1 also tries to pop in its own XP counter, I notice some discrepancies - like for example when I had only one PC fight 3 goblins, Assign XP tab tried to give him only 135 XP, and once I've cloned the PC into five PCs, the XP from same three goblins somehow increased to 500. 

![image](https://github.com/ironmonk88/monks-tokenbar/assets/158814704/243adee0-a5ca-4923-a9c9-b39100fbb1bd)

I can't seem to find any settings for adjusting the XP counting from the Token Bar. 
I'm on Foundry 11, build 325, Pathfinder 1e version 9.6, Token Bar version 11.11

**Comment by ironmonk88** [comment]
You're right, there was an issue... but I also think there's an issue with the PF1e code.
![image](https://github.com/ironmonk88/monks-tokenbar/assets/75920956/3dbbb3bc-f26e-4d83-9f16-2aaa82e2e809)
It sort of depends how you interpret the XP awards table.
They're calculating it as 3 Goblins @ 135XP = 405.  Divided by 5 players = 81 XP
And I'm going by the individual XP.  ie, for a party of 5, a goblin awards 35 XP to each player.  So 35 * 3 = 105 XP (It should show 525 xp since 105 * 5 = 525)

The PF1e code does not take into account that parties of 5 should be treated the same as a party of 4.

**Comment by Goldendice1** [comment]
Hey, I'd just like to say this is still an issue. More specifically, we're not given a choice between exact xp (what Kyle's advocating for) and abstract xp (what monk is referencing) - it forces you into abstract xp or completely undivided xp. https://aonprd.com/Rules.aspx?ID=253
